app/javascript/controllers/select_autocomplete_controller.js

```diff
@@ -0,0 +1,15 @@
+import { Controller } from "@hotwired/stimulus"
+import accessibleAutocomplete from 'accessible-autocomplete'
+
+// Connects to data-controller="select-autocomplete"
+export default class extends Controller {
+  static targets = [ "input" ]
+
+  connect() {
+    accessibleAutocomplete.enhanceSelectElement({
+      selectElement: document.querySelector(
+        `#${this.inputTarget.id}`
+        )
```

Comment:
Shouldn't it be possible to use the target directly? Rather than querying the document for the element that we already have available to us.

```suggestion
      selectElement: this.inputTarget
```